src/app/components/media/CreateMedia.js

```diff
@@ -64,7 +64,7 @@ class CreateMedia extends Component {
     };
 
     const onSuccess = (response) => {
-      const rid = response.createMedia.media.dbid;
+      const rid = response.createMedia.media.pm_dbid;
```

Comment:
I think we should replace the `CreateMediaMutation` by a `CreateProjectMediaMutation`